docs/eventing/event-delivery.md

```diff
@@ -63,7 +96,17 @@ The table below summarizes what delivery parameters are supported for each chann
 
 | Channel Type | Supported Delivery Parameters |
 | - | - |
+| GCP PubSub | none |
 | In-Memory | `deadLetterSink`, `retry`, `backoffPolicy`, `backoffDelay` |
 | Kafka | `deadLetterSink`, `retry`, `backoffPolicy`, `backoffDelay` |
 | Natss | none |
-| GCP PubSub | none |
+
+## Broker Support
+
+The table below summarizes what delivery parameters are supported for each Broker implementation.
+
+| Broker Class | Supported Delivery Parameters |
+| - | - |
```

Comment:
@pierDipi could you please add these lines for GCP Broker and its footnotes? Thank you!

```
| googlecloud | `deadLetterSink` [^1], `retry`, `backoffPolicy`, `backoffDelay` [^2] |

[^1]: deadLetterSink must be a GCP Pub/Sub topic uri:
    ```yaml
    deadLetterSink:
      uri: pubsub://dead-letter-topic
    ```

    Please see the
    [config-br-delivery](https://github.com/google/knative-gcp/blob/master/config/core/configmaps/br-delivery.yaml)
    ConfigMap for a complete example.

[^2]: The googlecloud broker only supports the `exponential` backoffPolicy.
```

Reply:
Done https://github.com/knative/docs/pull/2938/commits/3a11af50615c9a08caa4224923451fb410137e80

/cc @grantr

Reply:
Thanks @pierDipi! Looks like github doesn't support the footnote syntax I used. Oh well. Feel free to reformat however you want.